app/code/Magento/Cms/Test/Mftf/Test/AdminCreateCmsBlockTest.xml

```diff
@@ -42,4 +42,34 @@
         <actionGroup ref="SaveAndCloseCMSBlockWithSplitButtonActionGroup" stepKey="saveAndCloseAction" />
         <seeElement selector="div[data-role='grid-wrapper']" stepKey="seeGridPage" />
     </test>
-</tests>
\ No newline at end of file
+    <test name="AdminCreateCmsBlockWithMarginalSpaceTest">
```

Comment:
Hi @ajithkumar-maragathavel. Maybe I've missed this point in your previous PR but we were talking about the best practices and having one ActionGroup|Section|Test per file. As far as I see, you have moved the action group to a separate file and that's awesome. Could you do the same with the test, please? So we have a separate file with the single test in the result.

Thank you!

Reply:
@rogyar I will change the code and commit again. Thanks for reminding the best practices.
But I have one doubt regarding the best practices. I am looking at the existing MFTF code in the core. I can able to see a single test file contains multiple tests in it. 
Those files need to refactor right?

Reply:
@rogyar Made the changes and commit the code. Please review it.
Thanks for your suggestions.